### PR TITLE
code cheges in amazon = add example1 file

### PR DESCRIPTION
**Pull request by purusotham910**

### CODE REVIEW

File: amazon [hunk 1]
```diff
@@ -1,5 +1,6 @@
 this is bala' 
 
-Here is a list of best short stories for kids. KathaKids presents the best stories from Indian Mythology & Folklore. We present stories from cool characters like Tenali Raman and also from folktales like Panchatantra and Katha Sarit Sagar. We also have stories from Indian Mythology which includes Lord Krishna’s stories, stories from Ramayana & Mahabharata and stories of our lovable Gods Ganesha & Shiva.
+Wikipedia was one of the first spaces online where you could learn best stories from Indian Mythology & Folklore. We present stories from cool characters like Tenali Raman and also from folktales like Panchatantra and Katha Sarit Sagar. We also have stories from Indian Mythology which includes Lord Krishna’s stories, stories from Ramayana & Mahabharata and stories of our lovable Gods Ganesha & Shiva.
```
Comment (balakishore123) [inline amazon]: replace Wikipedia was one with  bala kishore

**Review by balakishore123 — COMMENTED** [review]
Please make changes in the comment

File: amazon [hunk 2]
```diff
@@ -1,5 +1,6 @@
 this is bala' 
 
-Here is a list of best short stories for kids. KathaKids presents the best stories from Indian Mythology & Folklore. We present stories from cool characters like Tenali Raman and also from folktales like Panchatantra and Katha Sarit Sagar. We also have stories from Indian Mythology which includes Lord Krishna’s stories, stories from Ramayana & Mahabharata and stories of our lovable Gods Ganesha & Shiva.
+bala kishore of the first spaces online where you could learn best stories from Indian Mythology & Folklore. We present stories from cool characters like Tenali Raman and also from folktales like Panchatantra and Katha Sarit Sagar. We also have stories from Indian Mythology which includes Lord Krishna’s stories, stories from Ramayana & Mahabharata and stories of our lovable Gods Ganesha & Shiva.
 
-Enjoy more than 150 stories on KathaKids!
\ No newline at end of file
+Enjoy more than 150 stories on KathaKids!
+We also have stories from Indian Mythology which includes Lord Krishna’s stories, stories from Ramayana & Mahabharata and stories of our lovable Gods Ganesha & Shiva.
```
Comment (balakishore123) [inline amazon]: remove empty spaces in code

Comment (purusotham910) [reply amazon]: space changes in amazon

**Pull request merged**